arangod/RestServer/DatabaseFeature.cpp

```diff
@@ -693,13 +693,11 @@ void DatabaseFeature::stop() {
   }
 #endif
 
-  for (auto& p : theLists->_databases) {
-    TRI_vocbase_t* vocbase = p.second;
-
+  auto stopVocbase = [](TRI_vocbase_t* vocbase) {
     // iterate over all databases
     TRI_ASSERT(vocbase != nullptr);
     if (vocbase->type() != TRI_VOCBASE_TYPE_NORMAL) {
-      continue;
+      return;
```

Comment:
why?

Reply:
The loop body was put into a lambda. Hence a continue becomes a return. The lambda is used twice below.

Reply:
True, for some reason, I thought you put cycle to lamda